tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -235,4 +237,231 @@ describe("E2E Testing for taqueria flextesa plugin",  () => {
+
+            // 2. Run taq simulate hello-tacos.tz
+            const {stdout, stderr} = await exec(`taq simulate ${dockerName} hello-tacos.tz --storage '5' --input '2'`, {cwd: `./${taqueriaProjectPath}`});
+
+            // 3. Verify that it's valid and contains no errors
+            expect(stdout).toBe(contents.oneRowTableSimulateResult);
+            expect(stderr).toBe("");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    });
+
+    test('Verify that taqueria simulator task will display proper message if user tries to simulate contract that does not exist', async () => {
+        try {
+            // 1. Run taq simulate ${contractName} for contract that does not exist
+            const {stdout, stderr} = await exec(`taq simulate ${dockerName} test.tz --storage '5' --input '2'`, {cwd: `./${taqueriaProjectPath}`})
+
+            // 2. Verify that output includes a table and an error message
+            expect(stdout).toBe(contents.nonExistent)
+            expect(stderr).toContain("Does not exist");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    });
+
+    test("Verify that taqueria simulator task emits logic error (runtime exception to be thrown)", async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts`)
+
+            // 2. Run taq simulate hello-tacos.tz
+            const {stdout, stderr} = await exec(`taq simulate ${dockerName} hello-tacos.tz --storage '5' --input '10'`, {cwd: `./${taqueriaProjectPath}`})
+
+            // 3. Verify that output includes a table and an error message
+            expect(stdout).toBe(contents.runtimeError)
+            expect(stderr).toContain("NOT_ENOUGH_TACOS");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    })
+
+    test("Verify that taqueria simulator task emits parameter type error (supplying string instead of nat)", async () => {
+        try {
+            // 1. Copy contract from data folder to taqueria project folder
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/contracts`)
+
+            // 2. Run taq simulate hello-tacos.tz
+            const {stdout, stderr} = await exec(`taq simulate ${dockerName} hello-tacos.tz --storage '5' --input '"hi"'`, {cwd: `./${taqueriaProjectPath}`})
+
+            // 3. Verify that output includes a table and an error message
+            expect(stdout).toBe(contents.runtimeError)
+            expect(stderr).toContain("unexpected string, only an int can be used here");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+    })
+
+    test("Verify that taqueria simulator task emits parameter type error (supplying list instead of nat)", async () => {
```

Comment:
I don't think this is strictly necessary since we have a test that tries a string and the error states that the input can only be an integer. I think only one test is needed since this isn't testing anything new

Reply:
Alternatively there are tests to specify a bad type value for input but not for storage. I think that one should also be covered and this test could be rewritten to cover that

Reply:
I was trying to test that the format to specify a list is valid. e.g. '{1;2;3}'

Reply:
Alright cool, my dyslexia hit and I didn't notice the work sequence was different from string in the previous test